<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarProdutos</class>
 <widget class="QDialog" name="AdicionarProdutos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabProdutos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>611</width>
     <height>471</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabIncluir">
    <attribute name="title">
     <string>Incluir</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>551</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCodigoIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrição:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDescricaoIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantidade:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtQtdeIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPrecoIncluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Preço unitário: R$</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIncluir">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Incluir</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPesquisar">
    <attribute name="title">
     <string>Pesquisar e/ou Editar</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>521</width>
       <height>391</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCodigoEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPesquisar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
     <widget class="QFrame" name="frDadosEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>501</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtDescricaoEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>361</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantidade:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtQtdeEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtPrecoEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Preço unitário: R$</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabExcluir">
    <attribute name="title">
     <string>Excluir</string>
    </attribute>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>501</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtCodigoExcluir">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcluir">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dado Excluido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDadosExcluidos">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>351</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabListar">
    <attribute name="title">
     <string>Lista de produtos</string>
    </attribute>
    <widget class="QTableWidget" name="twProdutos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>541</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMostrarLista">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Lista</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
